packages/integration_test/example/test_driver/extended_integration_test.dart

```diff
@@ -25,5 +25,5 @@ Future<void> main() async {
       }
       return true;
     },
-  );
+    writeResponseOnFailure: true);
```

Comment:
Hi @goderbauer, thanks for taking the time to review this PR. To my understanding, we cannot check whether the test has passed or if the file has been written out in `extended_integration_test.dart`. This is because the file is the --driver parameter in the `flutter drive` command and the `integrationDriver` function returns a Future<void>. At the end, it will call `exit(0)` or `exit(1)`, which terminates the `flutter drive` command. Therefore, we are unable to perform any checks after that point.

Reply:
Also if we verify file content, we have to verify in `dev/bots/test.dart`, as the write response function is totally same with `integration_test_driver.dart`( I steal from that driver), if you persist I am ok to update the code to check file content(both for  `integration_test_driver.dart` and `extended_integration_test_driver.dart`)

Reply:
I see. We should test that the file content is what is expected, though.

Reply:
hi, PR updated to check the file content